client/src/hooks/networks/utils.ts

```diff
@@ -0,0 +1,211 @@
+import {
```

Comment:
In this file, do you mind documenting the functions? Especially what they are used for.